docs/providers/aws/cli-reference/invoke.md

```diff
@@ -57,4 +75,31 @@ serverless invoke --function functionName --stage dev --region us-east-1 --path
 ```
 
 This example will pass the json data in the `lib/data.json` file (relative to the root of the service) while invoking
-the specified/deployed function.
\ No newline at end of file
+the specified/deployed function.
+
+#### Local function invocation
+
+```bash
+serverless invoke local --function functionName
+```
+
+This example will locally invoke your function.
+
+
+#### Local function invocation with event data
+
+You can input test data in `event.json` file inside your service directory:
+```json
+{
+  "foo": bar
+}
+
+```
+
+and then pass it with the command
+
+```bash
+serverless invoke local --function functionName -p event.json
```

Comment:
Maybe update `-p` to be verbose `--path` as the `--function` keyword is also verbose